.gitlab-ci.yml

```diff
@@ -179,6 +179,18 @@ batch_systems:
     - make test tests=src/toil/test/mesos/MesosDataStructuresTest.py
     - kill $(jobs -p) || true
 
+slurm_test:
+  image:
+    name: "quay.io/vgteam/vg_ci_prebake:latest"
+    entrypoint: ["/usr/bin/env"]
```

Comment:
Do we still need to override the image here if we aren't doing anything fancy in the script section anymore and we're instead just calling a full Bash script?

Because Lon changed the default image for the tests to be a Toil-specific prebake one with dependencies installed, and dropped the dependency install code from the Gitlab setup steps, so the image specified here might not work anymore.

```suggestion
```